src/LegendTextIO.jl

```diff
@@ -19,4 +19,21 @@ using RadiationDetectorSignals: group_by_evtno
 include("util.jl")
 include("geant4_csv.jl")
 
+## .root.hits files
+
+import Base, Tables
+
+using Mmap: mmap
+using Parsers
+
+export RootHitFile
+
+include("RootHitFile.jl")
+
+mutable struct Defaults
+	root_hits_batch_size::Int
+end
+
+const DEFAULTS = Defaults(10)
```

Comment:
I really like this way of defining defaults. It allows the user to change them if they want and it's easy to expand.

Reply:
Sure. Will do. :)